docs/new_viz_guidance.md

```diff
@@ -25,7 +25,7 @@ individual visualizations are imported and further organized below.
 
 ### Visualization Template
 
-[Visualization Template](https://github.com/oss-aspen/8Knot/blob/dev/pages/visualization_template/viz_template.py) gives you the outline to making your next visualization.
+The [Visualization Template](https://github.com/oss-aspen/8Knot/blob/dev/pages/visualization_template/viz_template.py) can take you step by step on creating your new visualization with boilerplate code with the overall structure and needs of a visualization.
```

Comment:
"takes you step-by-step through the process of creating a new visualization." 

The original sentence is a run-on. I think it's sufficient to point to the guide and say "Here's where you learn how to do the thing".